checkov/sca_image/runner.py

```diff
@@ -344,3 +328,6 @@ def get_image_details_from_twistcli_result(self, scan_result: dict[str, Any], im
 
     def included_paths(self) -> Iterable[str]:
         return ['.github', '.circleci']
+
+    def get_empty_report(self) -> Report:
+        return Report(self.check_type)
```

Comment:
I don't see the benefit to create a function for this one liner. Every function call in Python adds extra time and makes the code slower.

Reply:
thanks! I will fix it